google/ggrc-core
# GGRC-4744 Migrate audit scope documents to evidense phase 1

# Issue description

Please read details in the JIRA [issue](https://jira-7.corp.google.com/browse/GGRC-4744) description.
# Steps to test the changes

- Check migration on clean db, qa dump, etc
- Before migration. Create assessment with document assigned to it and reuse this document in related assessment. Run migration. Update document field e.g description and check if 2dn doc is not changed.
- Same way check ACL migration.
- Check revisions for migrated documents
- Check revisions for relationship at 'admin' -> 'events' 


# Solution description
Migration script was implemented, FE updated to prevent situations with multiple relationships during 'reuse' of documents from 'related assessments'

# Sanity checklist


# Migration checklist

**NOTE: no downgrade** for this migration as it too complicated.

# PR Review checklist
